frontend/src/app/modules/bim/ifc_models/bcf/list-container/bcf-list-container.component.ts

```diff
@@ -26,14 +26,37 @@ import {IfcModelsDataService} from "core-app/modules/bim/ifc_models/pages/viewer
 export class BcfListContainerComponent extends WorkPackageListViewComponent implements OnInit {
   @InjectField() bimView:BimViewService;
   @InjectField() ifcModelsService:IfcModelsDataService;
+  showTableView = false;
 
   public wpTableConfiguration = {
     dragAndDropEnabled: false
   };
 
+  ngOnInit() {
+    super.ngOnInit();
+
+    this.uIRouterGlobals
+          .params$!
+          .pipe(
+            this.untilDestroyed(),
+            pluck('cards'),
+            distinctUntilChanged(),
+            filter(cards => cards != null),
+          )
+          .subscribe((cards:boolean) => {
```

Comment:
Isn't that logic that belongs to a different PR? We do not introduce a table in the BCF module in this PR, right? Or is that simply a preparation?